docs/docs/guides/customization/texts.md

```diff
@@ -39,4 +39,4 @@ ZITADEL is available in the following languages
 
 A language is displayed based on your agent's language header. The default language is English.
 
-If you need support for a specific language we highly encourage you to [contribute translation files](/CONTRIBUTING.md) for the missing language.
+If you need support for a specific language we highly encourage you to [contribute translation files](../CONTRIBUTING.md) for the missing language.
```

Comment:
I think this needs to be an external link like https://github.com/zitadel/zitadel/blob/v2-alpha/CONTRIBUTING.md

Reply:
Out of context. This was due to a warning. Will try to remove it.